migrations/20170310092939_add_events_table.js

```diff
@@ -0,0 +1,30 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('events', table => {
+    table.string('id').primary().index();
```

Comment:
It's usually a good idea to have a numeric surrogate primary key field as it for example makes joins faster. If a field is needed to identify the row from application code, a separate field can be used. So summa summarum:

* Change the primary key to `table.increments('id').primary().index();`
* Add a separate `code` field `table.string('code').notNullable().unique().index();`